docs/Benchmarks.md

```diff
@@ -0,0 +1,94 @@
+
+This command line contains the specific version of the runtimes and SDK that were used, which makes it deterministic. This means that if you want to investigate a regression you can pick two points on a chart and re-run these to confirm the regression. You can also change each version to understand if the regression is coming from ASP.NET or the .NET Runtime.
+
+### Use local changes
+
+Using the same Crank command line you can also test the impact of local changes.
+
+- Build your changes in `release` for the desired architecture/os (if it matters).
+- Add `--application.options.outputFiles c:\build\release` and replace the folder by the one containing your changes.
+- Run the updated Crank command line
+
+## Benchmarking pull-requests
+
+Use this technique when you want to benchmark a community contributed PR or when you don't have the changes available locally.
+
+### Display all available benchmarks
+
+- In the pull-request, add `/benchmark` in the comments.
+- A text similar to this will be displayed (it might take up to 10 minutes):
+
+---
+Crank Pull Request Bot
+
+`/benchmark <benchmark[,...]> <profile[,...]> <component,[...]> <arguments>`
+
+Benchmarks:
+- `plaintext`: TechEmpower Plaintext Scenario - ASP.NET Platform implementation
+- `json`: TechEmpower JSON Scenario - ASP.NET Platform implementation
+- `fortunes`: TechEmpower Fortunes Scenario - ASP.NET Platform implementation
+- `yarp`: YARP - http-http with 10 bytes
+- `mvcjsoninput2k`: Sends 2Kb Json Body to an MVC controller
+
+Profiles:
+- `aspnet-perf-lin`: INTEL/Linux 12 Cores
+- `aspnet-perf-win`: INTEL/Windows 12 Cores
+- `aspnet-citrine-lin`: INTEL/Linux 28 Cores
+- `aspnet-citrine-win`: INTEL/Windows 28 Cores
+- `aspnet-citrine-ampere`: Ampere/Linux 80 Cores
+- `aspnet-citrine-amd`: AMD/Linux 48 Cores
+
+Components:
+- `kestrel` (Microsoft.AspNetCore.Server.Kestrel `src\Servers\Kestrel\build.cmd`)
+- `mvc` (Microsoft.AspNetCore.Mvc `src\Mvc\build.cmd`)
+- `routing` (Microsoft.AspNetCore.Routing `src\Http\build.cmd`)
+
+See [prbenchmarks.aspnetcore.config.yml](https://github.com/aspnet/Benchmarks/blob/2bddb9b43ffbc8cfdb4a2c3777d4e5213bc42332/build/prbenchmarks.aspnetcore.config.yml) for more details.
+
+Arguments: any additional arguments to pass through to crank, e.g. `--variable name=value`
+
+---
+
+Where:
+- *Benchmark* is the application that will be stressed.
+- *Profile* is the environment where the application will run on.
+- *Component* is the ASP.NET project under test that will be built and contains your changes.
+
+> [!IMPORTANT]  
+> Any changes outside of the "component" projects will not be benchmarked.
+
+Create a new comment with the correct benchmarks, profiles, and components to build. For instance:
+
+```console
+/benchmark json aspnet-citrine-lin kestrel
+```
+
+After up to 10 minutes a link should be displayed. Currently the results are not added to the PR anymore, you will need to follow the link and copy the results when the benchmark is finished.

```

Comment:
how do I know whether the results represent a regression? does it automatically do a baseline on HEAD at the same time? or i have to go compare/interpret the dashboard (that seems error prone given the complexity of the dashboard)

does it help me understand whether a diff might  be noise?